packages/integration-tests/tests/ui/withdraw.spec.ts

```diff
@@ -0,0 +1,104 @@
+import { expect, test } from "@playwright/test";
+
+import { config } from "./config";
+import { BlockExplorer, Buffer, Wallets } from "../../src/entities";
+import { Helper } from "../../src/helper";
+
+const bufferRoute = "src/playbook/";
+const helper = new Helper();
+let url: string;
+let bufferFile;
+let withdrawTxHash: string;
+let fromAddress: string;
+
+//@id1661
+test("Check Withdraw ETH transaction on BE", async ({ page }) => {
+  bufferFile = bufferRoute + Buffer.txEthWithdraw;
+  withdrawTxHash = await helper.getStringFromFile(bufferFile);
+  url = BlockExplorer.baseUrl + `/tx/${withdrawTxHash}` + BlockExplorer.localNetwork;
+  fromAddress = Wallets.richWalletAddress;
+  console.log(url);
+
+  await page.goto(url);
+
+  const hash = await page.locator(`//*[text()="Transaction Hash"]/..//..//*[text()="${withdrawTxHash}"]`).first();
+  const fromAddressElement = await page.locator(`text=${fromAddress}`).first();
+  const toAddressElement = await page
+    .locator(`//*[text()="To"]/..//..//..//*[text()="L1"]/..//*[text()="0x36615Cf349d...c049"]`)
+    .first();
+  const ethValue = await page.locator(`text=0.000009`).first();
```

Comment:
we have this value in many places and AFAK Oleh tries to refactor it to move to entities.ts. I would suggest to contact with him and use a variable instead of the value 0.000009 .